### ISSUE TITLE
[Defacepocalypse] Convert Spree products#new views to OFN views

### ISSUE THREAD

**Issue by sauloperez** [openfoodfoundation/openfoodnetwork#3049]
## Description

The backoffice's new product page is heavily Defaced which makes very hard to follow and to find out where things are defined.

This has been the problem in https://github.com/openfoodfoundation/openfoodnetwork/issues/2982. There's a lot of indirection due to OFN views + Spree views + Deface and so there's no way to foresee where that error comes from (without spending lots of hours).

As of today, this is the list of overrides I could find involving this page:

```shell
$ ll app/overrides/spree/admin/products/new  
total 8.0K
-rw-r--r-- 1 pau pau 4.9K Nov 15 10:36 replace_form.html.haml.deface
$ ll app/overrides/spree/admin/products/_form 
total 44K
-rw-r--r-- 1 pau pau  358 Nov 14 17:04 add_description_wysiwyg.html.haml.deface
-rw-r--r-- 1 pau pau  106 Nov 14 17:04 add_primary_taxon_field.html.haml.deface
-rw-r--r-- 1 pau pau  300 Nov 14 17:04 add_supplier.html.haml.deface
-rw-r--r-- 1 pau pau 1.1K Nov 14 17:04 add_units_form.html.haml.deface
-rw-r--r-- 1 pau pau  131 Nov 14 17:04 remove_available_on.deface
-rw-r--r-- 1 pau pau  121 Nov 14 17:04 remove_cost_currency.deface
-rw-r--r-- 1 pau pau  118 Nov 14 17:04 remove_cost_price.deface
-rw-r--r-- 1 pau pau   49 Nov 14 17:04 remove_meta_description.deface
-rw-r--r-- 1 pau pau   48 Nov 14 17:04 remove_option_types_and_taxons.deface
-rw-r--r-- 1 pau pau  172 Nov 14 17:04 replace_master_price_label.html.haml.deface
-rw-r--r-- 1 pau pau  197 Nov 14 17:04 replace_taxons_div.html.haml.deface
```

## Expected Behavior

The goal to have all these bits and pieces together as OFN views and partials as any regular Rails app. This will make it easier to change things in the feature, which we will much appreciate in the Spree upgrade.

**Comment by luisramos0** [comment]
@HugsDaniel @sauloperez the 2 issues are closed, is there something to be done in master about this one or can we close this epic?

**Comment by HugsDaniel** [comment]
I don't think there is, the rest will be done in 2-0 I guess !
